src/templates/partials/footer.html

```diff
@@ -1,42 +1,14 @@
-<div id="directory-footer" class="main full c">
-  <div class="group">
-    <a class="button devhub" href="/developers/">{{ _('Develop Apps') }}</a>
-  </div>
-  <div class="group links">
-    <a href="/developers/">{{ _('Developer Hub') }}</a>
-    <a href="#" class="submit-feedback">{{ _('Feedback') }}</a>
-    {% if capabilities.firefoxOS %}
-      <a href="https://support.mozilla.org/products/firefox-os/marketplace" target="_blank">{{ _('Support') }}</a>
-    {% elif capabilities.firefoxAndroid %}
-      <a href="https://support.mozilla.org/kb/install-android-apps-using-marketplace" target="_blank">{{ _('Support') }}</a>
-    {% endif %}
-  </div>
-  <div class="language group links">
-    {# <a href="{{ url('purchases') }}">{{ _('My Apps') }}</a> #}
-    {% if user.get_permission('developer') %}
-      <a href="/developers/submissions" rel="external">{{ _('My Submissions') }}</a>
-    {% endif %}
-    {% if user.get_permission('reviewer') %}
-      <a href="/reviewers/apps/queue/" rel="external" class="only-logged-in">{{ _('Reviewer Tools') }}</a>
-    {% endif %}
-    {% if user.get_permission('lookup') %}
-      <a href="/lookup/" rel="external" class="only-logged-in">{{ _('Lookup Tool') }}</a>
-    {% endif %}
-  </div>
-</div>
-<div id="footer">
-  <div class="pad">
-    <h1 id="footzilla"><a href="https://www.mozilla.org/" target="_blank">mozilla</a></h1>
-    <p>
-      {{ _('Except where otherwise <a href="{legal_url}" target="_blank">noted</a>, content on this site is licensed under the <a href="{cc_url}" target="_blank">Creative Commons Attribution Share-Alike License v3.0</a> or any later version.',
-           legal_url='http://www.mozilla.org/about/legal.html#site',
-           cc_url='http://creativecommons.org/licenses/by-sa/3.0/') }}
-    </p>
-    <ul>
-      <li><a href="https://marketplace.firefox.com/privacy-policy">{{ _('Privacy Policy') }}</a></li>
-      <li><a href="https://marketplace.firefox.com/terms-of-use">{{ _('Terms of Use') }}</a></li>
-      <li><a href="https://www.mozilla.org/legal/fraud-report/index.html" target="_blank">
-        {{ _('Report Trademark Abuse') }}</a></li>
-    </ul>
-  </div>
-</div>
+<h1><a href="https://www.mozilla.org/" target="_blank">mozilla</a></h1>
```

Comment:
This will ultimately end up in marketplace-template, other projects can pick it up at their leisure. Just threw something quick together for now while pwalm is busy.